tremor-cli/build.rs

```diff
@@ -0,0 +1,43 @@
+use clap::IntoApp;
+use clap_mangen::Man;
+use std::{
+    env,
+    fs::File,
+    io::Error,
+    path::{Path, PathBuf},
+};
+
+include!("src/cli.rs");
```

Comment:
Do we really want to copy in the entire source here?

Reply:
What else would you do? We can't `use` our code from the `src` dir, but we got to have the definition at hand somehow. I don't see another way and I don't dislike this one.

Reply:
Yeah, I couldn't find another way of importing the Cli object because we need it to generate the man-page, so I did it this way

Reply:
But is there another way of doing it?